docs/architecture/data.md

```diff
@@ -111,3 +113,46 @@ class first_gcs,second_gcs,validated_gcs gcs_group_boxstyle
 class first_bq,second_bq bq_group_boxstyle
 class gcs_label1,gcs_label2,gcs_label3,bq_label1,bq_label2 group_labelstyle
 ```
+
+## Adding a new data source
+
+Adding a new data source based on the architecture described above involves several steps, outlined below.
+
+```{note}
+If you're bringing in data that is similar to existing data (for example, a new subset of an existing dataset like a new Airtable or Littlepay table), you should follow the existing pattern for that dataset. [This spreadsheet](https://docs.google.com/spreadsheets/d/1bv1K5lZMnq1eCSZRy3sPd3MgbdyghrMl4u8HvjNjWPw/edit#gid=0) gives overviews of some prominent existing data sources, outlining the specific code/resources that correspond to each step in the [general data flow](data-ingest-diagram) for that data source.
```

Comment:
Are permissions correct on the spreadsheet for future use? And is ownership of that file shared with somebody whose Google account is not at risk of future deletion?

Reply:
It's saved in the Cal-ITP Data Services Google Drive